README.md

```diff
@@ -1,23 +1,30 @@
+* **Services** - As of today, Jazz can help build functions, APIs and static websites. Fully customizable template-based design makes it easy for admins to define new ones and expose them as services to the developers. Support for [AWS SAM](https://github.com/awslabs/serverless-application-model) & [Serverless Framework](https://github.com/serverless/serverless) templates will be coming soon - this will allow developers to define their serverless applications using these commonly used frameworks!
+* **Deployment Targets** - Currently, Jazz can deploy to AWS serverless services including Lambda, API Gateway, S3, CloudFront, DynamoDB, SQS, Kinesis etc. We plan to support related services in Azure and GCP in the near future.
+* **CI/CD** - Jazz comes with CI/CD by default. It creates a code repository for each service, assigns appropriate permissions and adds a web hook to trigger build/deployment workflows whenever it sees a commit. We leverage [Jenkins](https://github.com/jenkinsci/jenkins) open source for the build process and [Serverless Framework](https://github.com/serverless/serverless) for deploying these services.
+* **One-Click Development Environments** - Jazz automatically creates a new dedicated development environment for each git branch in the service repository. This allows parallel development in a multi-developer team without a hitch! Jazz takes care of deleting these environments once the branch gets deleted.
+* **Security Controls** - Centralized platform allows administrators to define & apply security controls from a single place through Jazz. Admins can choose to enforce the controls on every service that gets created using Jazz allowing them to make them secure by default. Many best practices like applying the principle of least privilege, code (& dependency) scans during CI/CD, preventing default public access are available by default in Jazz.
+* **Extensions** - Jazz is designed to integrate and work well with other systems that your enterprise needs. You can check out our optional extensions like Slack, Splunk, Apigee etc. Centralized configuration helps Jazz admins to easily enable/disable these features as per their needs.
+* **Multiple Accounts & Regions** - Jazz's multi account/region extension allows you to add support for multiple accounts & regions to an existing Jazz stack. Developers can simply select the target account & region from the UI to deploy their services. Admins can add/remove new accounts/regions to the stack with ease.
+* **Other Features** - Jazz comes with many useful integrations like SCM (Gitlab/Bitbucket), monitoring (CloudWatch), logging (ElasticSearch), authentication (Cognito), access controls, code quality metrics (SonarQube) by default.
+* **Utilities** - Jazz comes many utilties that allow admins to monitor, govern and report the usage of serverless services through Jazz environment.
```

Comment:
Should read: Jazz comes with many utilities (typo and missing with)

Reply:
Done.